src/tests/operate.test.js

```diff
@@ -0,0 +1,38 @@
+import Big from 'big.js';
+import operate from '../components/logic/operate';
```

Comment:
[REQUIRED] The project requirements require that there's the usage of `React Testing Library` to simulate user interaction.

React Testing Library is a tool that helps you test user interactions with your React components. It provides methods to simulate user interactions with different interactive elements such as buttons, input fields, and select elements.

For example, you can use the `userEvent` library to simulate a user clicking on a button. You can then test how your component behaves in response to these simulated interactions.
Here is an example test for the `Button` component that is rendered in the `Calculator` component:

```
import React from 'react';
import { render, screen } from '@testing-library/react';
import userEvent from '@testing-library/user-event';
import Calculator from './Calculator';

test('Button click updates display value', () => {
  render(<Calculator />);
  const button = screen.getByText('7');
  const display = screen.getByText('0');
  userEvent.click(button);
  expect(display.textContent).toBe('7');
});
```

This test renders the `Calculator` component and uses the `getByText` method from `screen` to find the button with the text `'7'`. It then uses the `userEvent.click` method to simulate a user clicking on the button. Finally, it checks that the text content of the display element has been updated to `'7'`.